src/MovieCard/MovieCard.js

```diff
@@ -177,7 +184,9 @@ class MovieCard extends Component {
           <section className="rating-section">
             <p>
               {this.state.foundRating ? (
-                <b className="user-rating-msg">You rated {this.state.foundRating} </b>
+                <b className="user-rating-msg">
+                  You rated {this.state.foundRating}{" "}
```

Comment:
what's the purpose for the extra {" "} on this line ?

Reply:
That's a mistake, not sure how that ended up there